lib/client.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+# Basic client contract. All other clients (blockchain.info, client with fallback)
+# must implement this methods)
+# @todo #36:30m Let's create some implementations of this class.
+#  For now, this class doesn't have any implementation and
+#  its main purpose is a contract that must be implemented
+#  by clients with different APIs and functionality.
+# @todo: #36:30m Needs to implement a mechanism for getting API by its core URL or name.
+#  For example, the Clients class has a method client(name) that returns the proper client.
+class Client
```

Comment:
@aivinog1 what is the point of having this class? There are no interfaces in Ruby. We won't gain anything by introducing this class.